src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue

```diff
@@ -1,15 +1,129 @@
 <template>
-  <div>
-    Future page for Travel Estimates
-  </div>
+  <v-container fluid class="container-max-width">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">
+          How do you want to estimate your travel needs?
+        </h1>
+        <p class="page-intro">
+          Consider the travel requirements that you previously outlined (e.g.,
+          location, duration, quantity, attendance, etc.). You may choose to set
+          a ceiling price and apply it to all periods. Or, if you know that your
+          travel requirements will change over time, then you may opt to set a
+          different price for each period.
+        </p>
+        <div class="copy-max-width">
+          <ATATRadioGroup
+            class="copy-max-width mb-10 max-width-740"
+            id="TravelEstimates"
+            :card="true"
+            :items="travelEstimateOptions"
+            @click="travelFormFields"
+            :value.sync="selectedTravelEstimate"
+            :rules="[$validators.required('Please select an option')]"
+          />
+        </div>
+
+      
```

Comment:
Missed this the first time around - the `hr` is missing between the radio group and inputs
![Screen Shot 2022-10-06 at 10 39 56 AM](https://user-images.githubusercontent.com/90333349/194342728-4792b8ac-9900-44de-b1ea-01a1e26e302d.png)